src/fxcorrelator_bengops.py

```diff
@@ -1355,13 +1363,13 @@ def spead_issue_all(corr, beams=[], from_fpga=True):
     :param from_fpga:
     :return:
     """
-    spead_static_meta_issue(corr, beams=beams)
+    spead_static_meta_issue(corr, beams=beams)  # fxcorrelator.spead_issue_meta
```

Comment:
That info if beamformer is there or not should be in the ini file? For example check first if [beamformer] field exists?